src/game/containers/RoundContainer.js

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact';
+import { connect } from 'preact-redux';
+
+import { RoundLayout } from '../components/RoundLayout';
+import { Countdown } from '../../components/Countdown/Countdown';
+
+import * as RoundPhase from '../../shared/constants/round-phase';
+import { DURATION } from '../../shared/constants/round';
+
+const PureRound = ({ roundPhase, timeRemaining }) =>
+    roundPhase !== RoundPhase.IN_PROGRESS && roundPhase !== RoundPhase.FINISHED ? null :
+
+    <RoundLayout>
+        <Countdown
+            timeAmount={DURATION}
+            timeRemaining={timeRemaining}
+        />
+    </RoundLayout>
```

Comment:
I see SelectorInput in src/game/components/RoundLayout.js.
Should SelectorInput be added here?

Reply:
I want to add it alongside with redux stuff. Layout node is just a placeholder for future PR